guides/theme-quality-assurance.md

```diff
@@ -0,0 +1,355 @@
+
+Much of this test and QA process can be automated. However, there is great value in human experience, so we recommend a **real** human being verify all final quality tests before launch.
+
+---
+
+##### Table of Contents
+{:.no_toc}
+
+* An unordered list to be replaced with the ToC, excluding the "##### Table of Contents" from above
+{:.no_toc}
+{:toc levels=1}
+
+---
+
+## Testing Environments
+
+To maximize ecommerce conversion rates, verify these tests in the following browser environments:
+
+### Desktop
+{:.no_toc}
+
+* Windows
+  * Firefox 60+
+  * Chrome C66+
+  * IE 11+
+  * Edge 16+
+  * Opera 53+
+* Mac
+  * Safari 10+
+  * Chrome C66+
+  * Firefox 60+
+  * Opera 53+
+
+### Tablet
+{:.no_toc}
+
+* Android
+  * Current Galaxy
+  * Marketshare Winner
+* iOS
+  * Current iPad Pro
+  * Current iPad
+
+### Mobile
+{:.no_toc}
+
+* Android
+  * A currently-leading Samsung Galaxy phone
+  * A currently-leading Google Pixel phone
+* iOS
+  * A currently-leading large iPhone
+  * A currently-leading small iPhone
+
+*Note: the needs of every client are unique, so thoroughly research and test the environments used most by their target customers in addition to those above.*
+
+---
+
+## Theme Sections
+
+Verify each of the following stories are true:
+
+### Home Page/Overall
+{:.no_toc}
+
+* Home page should be width-responsive and should behave as expected at phone (landscape and portrait), tablet (landscape and portrait), and desktop resolutions and aspect ratios
+* Products visible on the Home page should link to product detail pages
+* Home page navigation should be present, usable, and consistent throughout the site
+* Client’s brand colors should be ubiquitous and consistently accurate throughout the site
+* Typography should be legible and styled in a typeface representative of the client’s brand
+* Meta data and other social and SEO structures should be present and well-formed
+* Tracking scripts should track views of the home page correctly
+* Home page tap/click targets should visible, large enough to be tapped/clicked, and unobscured by the edges of the viewport window
+* Body copy should be engaging, grammatically correct, and consistent with client's brand message
+* Animations and JavaScript UX enhancements should be lightweight, easy to understand, and should fail gracefully in both incompatible and compatible browsers – use them to enhance, but don’t rely on them
+
+### Navigation
+{:.no_toc}
+
+* Navigation should be width-responsive and should behave as expected at phone (landscape and portrait), tablet (landscape and portrait), and desktop resolutions and aspect ratios
+* Navigation should be present, usable, and consistent throughout the site
+* Navigation tap/click targets should visible, large enough to be tapped/clicked, and unobscured by the edges of the viewport window
```

Comment:
As I mentioned above, I believe that there is a "be" that has been left out of this sentence - "...targets should be visible...". 🐝